nrf_modem/Kconfig

```diff
@@ -12,14 +15,24 @@ choice NRF_MODEM_BUILD_STRATEGY
 
 config NRF_MODEM_LINK_BINARY
 	bool "Link binary"
+	select NRF_MODEM_LINKABLE
 endchoice
 
-config NRF_MODEM_LOG
-	depends on NRF_MODEM_LINK_BINARY
-	bool "Use version of the library with logs"
+choice NRF_MODEM_BINARY_CHOICE
+	prompt "Choose binary"
+	depends on NRF_MODEM_LINKABLE
+	default NRF_MODEM_BINARY_DEFAULT
+
+config NRF_MODEM_BINARY_DEFAULT
+	bool "default"
 	help
-	  Links the application with the library version capable of emitting logs.
-	  This increases the final size of the application.
+	  Link with the default binary.
+
+config NRF_MODEM_BINARY_LOG
```

Comment:
I'm wondering whether we should keep NRF_MODEM_LOG to enable modem library logs(?). This changes the Kconfig option to enable library logs and requires an update to the documentation.

Reply:
Would it makes sense to keep NRF_MODEM_LOG and have that select the binary version?

Reply:
Yes, it would. That's a good point, I will change that.